packages/core/src/components/SupportButton/SupportButton.tsx

```diff
@@ -54,13 +46,42 @@ const useStyles = makeStyles(theme => ({
   },
 }));
 
-export const SupportButton = ({
-  slackChannel = '#backstage',
-  email = [],
-  children,
-}: // plugin,
-PropsWithChildren<Props>) => {
-  // TODO: get plugin manifest with hook
+const SupportIcon = ({ Icon }: { Icon: IconComponent }) => <Icon />;
+
+const SupportLink = ({ link }: { link: SupportItemLink }) => (
+  <Link to={link.url} target="_blank">
```

Comment:
i like to add `rel="noreferrer noopener"` to external links - this is something we'll be adding to the Link class eventually but it's not there yet